app/frontend/styles/_application-card.scss

```diff
@@ -21,13 +21,13 @@
 
   .govuk-grid-column-one-third {
     @include govuk-media-query($from: tablet) {
-      width: 37%;
+      width: 40%;
```

Comment:
As this is used across the app, is there any reason why it is changed as part of this PR?

Reply:
I agree, maybe we can add an override to this under a new class name, similar to this, and append that to the application page specifically?

Reply:
This change is in `_application-card.scss`, and under the `.app-application-card` class. It will only affect elements that are already classified as application cards. It's going to be a bit tougher to try and scope this only for specific instances of the application card component.

Reply:
Sorry about the mirsunderstanding. The generic component name made me think this was reusable across a number of forms!

Reply:
ah oops awesome, this is exactly what we need 👍